cmd/dispatcher/main.go

```diff
@@ -41,14 +41,15 @@ type envConfig struct {
 	// Should failed deliveries be requeued in the RabbitMQ?
 	Requeue bool `envconfig:"REQUEUE" default:"false" required:"true"`
 
+	// Number of concurrent messages in flight
+	PrefetchCount int           `envconfig:"PREFETCH_COUNT" default:"10" required:"false"`
 	Retry         int           `envconfig:"RETRY" required:"false"`
 	BackoffPolicy string        `envconfig:"BACKOFF_POLICY" required:"false"`
 	BackoffDelay  time.Duration `envconfig:"BACKOFF_DELAY" required:"false"`
 }
 
 const (
 	defaultBackoffDelay = 50 * time.Millisecond
-	defaultPrefetch     = 1
 	defaultPrefetchSize = 0
```

Comment:
I would remove this constant and set it directly in the `channel.Qos/3` function call.